rllib/agents/trainer_config.py

```diff
@@ -902,15 +902,23 @@ def offline_data(
                 when the offline file has been generated by another RLlib algorithm
                 (e.g. PPO or SAC), while "normalize_actions" was set to True.
             input_evaluation: DEPRECATED: Use `off_policy_estimation_methods` instead!
-            off_policy_estimation_methods: Specify how to evaluate the current policy.
+            off_policy_estimation_methods: Specify how to evaluate the current policy,
+                along with any optional config parameters.
                 This only has an effect when reading offline experiences
                 ("input" is not "sampler").
-                Available options:
-                - "simulation": Run the environment in the background, but use
+                Available keys:
+                - {"simulation": None}: Run the environment in the background, but use
```

Comment:
I'm actually not sure about this option right now. Do we still need it?
If we say that these estimator methods should - preferrably - be run on the evaluation worker track, and the user wanted to just see the live-env evaluation (actual rewards), then we wouldn't need estimators, but just do config.input_ = "sampler" on the evaluation workers, no?

Reply:
I'd say let's keep it for now, and I'll fix it in the next PR when I actually move the OPE to the eval workers?